manifests/overlays/authentication/deployment.yaml

```diff
@@ -5,12 +5,14 @@
     args:
       - --https-address=:8443
       - --provider=openshift
-      - --openshift-service-account=odh-dashboard
+      - --client-id=odh-dashboard
+      - --client-secret=zPI1OE6GuN2o_PfmjhN0itZk7VWLdmHEsh6LMRrOw1Q
```

Comment:
Find a way of not hardcoding this secret here.

Reply:
Similar to https://github.com/opendatahub-io/odh-dashboard/pull/590 and use `--client-secret-file`